internal/pkg/policy/parsed_policy_test.go

```diff
@@ -514,13 +506,462 @@ func TestNewParsedPolicy(t *testing.T) {
 		if pp.Default.Name != "other" {
 			t.Error("other output should be identified as default")
 		}
-		if pp.Default.Role == nil {
+		defaultOutput := pp.Outputs[pp.Default.Name]
+		if defaultOutput.Role == nil {
 			t.Error("other output role should be identified")
 		}
 
 		expectedSha2 := "d4d0840fe28ca4900129a749b56cee729562c0a88c935192c659252b5b0d762a"
-		if pp.Default.Role.Sha2 != expectedSha2 {
-			t.Fatal(fmt.Sprintf("Expected sha2: '%s', got '%s'.", expectedSha2, pp.Default.Role.Sha2))
+		if defaultOutput.Role.Sha2 != expectedSha2 {
+			t.Fatal(fmt.Sprintf("Expected sha2: '%s', got '%s'.", expectedSha2, defaultOutput.Role.Sha2))
 		}
 	}
 }
+
+const noESOutputPolicy = `
```

Comment:
Can we move that into an external file, This would make json linting and updating it easier?